content/en/agent/guide/gcp-private-service-connect.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Connect to Datadog over GCP Private Service Connect
+kind: guide
+further_reading:
+    - link: '/integrations/google_cloud_platform/'
+      tag: 'Documentation'
+      text: 'Datadog-Google Cloud Platform Integration'
+    - link: '/agent/guide/private-link'
+      tag: 'Documentation'
+      text: 'Connect to Datadog over AWS PrivateLink'
+---
+
+{{% site-region region="us,us3,eu,gov,ap1" %}}
+<div class="alert alert-warning">This feature is not supported for the selected Datadog site.</div>
+{{% /site-region %}}
+
+{{% site-region region="us5" %}}
+
+[Private Service Connect][1] (PSC) allows you to connect directly to Datadog without using the public internet.
+
+Datadog exposes some of its data intake services in Google Cloud Platform as PSC [_published services_][2], as seen in the [table of published services](#published-services). 
+
+You can configure a GCP PSC endpoint to expose a private IP address for each Datadog intake service; this IP address routes traffic to the Datadog backend. You can then configures a GCP [_Private DNS Zone_][3] to override the DNS names corresponding to the products for each endpoint that is consumed. 
+
+{{< img src="agent/guide/psc/gcp-psc-overview-1.png" alt="GCP Private Service Connect schema. On the left, a 'Customer VPC' box contains Datadog Agents sending data to a PSC endpoint. On the right, a 'Datadog VPC' box contains a service attachment in communication with Datadog services. The PSC endpoint in the 'Customer VPC' box connects to the service attachment in the 'Datadog VPC' box through the GCP Private Backbone. " >}}
+
+## Setup
+
+### Connect an endpoint
+
+1. In your GCP console, navigate to **Network services** > **Private Service Connect**.
+2. Go to the **Endpoints** section. Click on **Connect endpoint**.
+   {{< img src="agent/guide/psc/connect-endpoint.png" alt="Screenshot of a 'Connect endpoint' page in the GCP console" >}}
```

Comment:
I agree with your point about needing the screenshot of the external site for the instructions to make sense.